src/Biblys/Isbn/Isbn.php

```diff
@@ -353,6 +381,11 @@ private function setAgency($agency)
         $this->_agency = $agency;
     }
 
+    private function setPrefix($prefix)
+    {
+        $this->_prefix = $prefix;
+    }
+
```

Comment:
On second thought, could you rename the method and property everywhere to something like `gtin14Prefix` so there's not confusion with xml ranges prefixes or any future other kind of prefix?

Reply:
Absolutely – done!